WordPress/src/main/res/layout/quick_start_list_item.xml

```diff
@@ -52,7 +52,6 @@
                 android:layout_marginEnd="@dimen/margin_extra_large"
                 android:layout_marginStart="@dimen/margin_large"
                 android:contentDescription="@string/quick_start_list_task_complete"
-                android:src="@drawable/ic_checkmark"
```

Comment:
I couldn't find the drawable in the trunk branch, anybody know where the file is?

Reply:
Not sure how this icon is missing 🤔 
In any case I think we could use `ic_checkmark_white_24dp` instead

<img src="https://github.com/wordpress-mobile/WordPress-Android/assets/304044/525c3277-ef1c-48d1-a9d4-a6134a0b1e81" width=360 />

Reply:
Not sure why the missing resources suddenly appeared today, they are contained in the module of `stories 2.4.0`. And the alternatives you suggested are the correct icons, thanks a lot!